docs/getting-started-guides/kubeadm.md

```diff
@@ -8,12 +8,24 @@ li>.highlighter-rouge {position:relative; top:3px;}
 ## Overview
 
 This quickstart shows you how to easily install a secure Kubernetes cluster on machines running Ubuntu 16.04 or CentOS 7.
-The installation uses a tool called `kubeadm` which is part of Kubernetes 1.4.
+The installation uses a tool called `kubeadm` which is alpha as of Kubernetes 1.4.
 
 This process works with local VMs, physical servers and/or cloud servers.
 It is simple enough that you can easily integrate its use into your own automation (Terraform, Chef, Puppet, etc).
 
-**The `kubeadm` tool is currently in alpha but please try it out and give us [feedback](/docs/getting-started-guides/kubeadm/#feedback)!**
+kubeadm is not a complete provisioning tool; it is designed as a part of a larger provisioning
```

Comment:
"provisioning" is a really squishy word.  I would say something like this and merge it with the paragraph above:

"""
`kubeadm` doesn't do all the work to create a cluster from scratch.  Instead, it is assumed that the user already has a set of machines (virtual or real) that are up and running.  A combination of native Linux distribution packages and container images are used to bootstrap Kubernetes on those machines.  `kubeadm` is built to be able to integrate easily with other lower level automation (Terraform, Chef, Puppet, etc.).
"""